src/aeso_ast_infer_types.erl

```diff
@@ -2016,7 +2048,8 @@ next_count() ->
 
 ets_tables() ->
     [options, type_vars, constraints, freshen_tvars, type_errors,
-     defined_contracts, warnings, function_calls, all_functions].
+     defined_contracts, warnings, function_calls, all_functions,
+     ord_constraint_types].
```

Comment:
There is no such table as I didn't find a use for it, if all `id`s are comparable by equality, what should go in that table?

Reply:
I don't get why there is a need for a table for `ord` but not for `eq`. Would you explain why one is needed and the other is not?

Reply:
Since all types of the form `{id, _, Id}` are compared by `eq`, there is no need to store all of them in a table, but not all types of the above form are compared by `ord` so I only need to store that.

So checking if `{id, _, "int"}` is comparable by `ord` will only succeed because `"int"` is in the table `ord_constraint_types`, but checking if any other type of the form `{id, _, Id}` is comparable by `eq` will always succeed.

Reply:
Check this commit that made `address` comparable by `ord` fc2875965e302c75a1c02e9655beeb0c524aecee.